client/src/components/common/applicant-info-1.vue

```diff
@@ -321,7 +321,7 @@
                 class="mt-2 pt-0"
               />
             </v-col>
-            <ApplicantInfoNav :isValid="isValid" />
+            <ApplicantInfoNav @nextAction="nextAction" />
```

Comment:
If possible add () to method (to make it clear to the reader that it's a method).